CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.3.1]
+* Add support for QR+PIN (#1929)
```

Comment:
Can you remove this for now? Add something more generic, like framework to return preferred auth method from iOS broker?